### ISSUE TITLE
No HTML markup for programmatically created pages

### ISSUE THREAD

**Issue by arielk5740** [gatsbyjs/gatsby#7648]
Hi,
I have a gatsby site with programmatically created pages (integration with drupal backend). 

The problem with such pages that affects SEO - there is no rendered content to HTML source,
meaning it has <div id="___gatsby"></div> only and no HTML markup.
It looks like the "demo" project (https://using-drupal.gatsbyjs.org/recipes/2/) has the same issue.

Any solution for this?
Thanks

**Comment by btk** [comment]
What do you mean "no HTML markup"? 

The link you supplied is just a classic html page with html tags inside the div you mention.

Try on your console;
```
curl https://using-drupal.gatsbyjs.org/recipes/2/ > drupalpage.html
```

It is just a normal HTML page with 100% SEO compatible.

**Comment by arielk5740** [comment]
@btk I meant no HTML tags on "view page source" in google chrome.
Regarding the demo site, you are right,  CURL indeed returns the full HTML.
But for my project both of CURL / "view page source" returns the structure like:

`<body><noscript>Please enable javascript in your browser</noscript><div id="___gatsby"></div><<script>/*<![CDATA[*/["/commons-1535269777918-33ca991fafeba7713c00.js","/app-1535269777918-7807a8f8daaa70be8e72.js","/path---test-page-1535269777918-e0cf04c7d606666eaa90.js","/component---src-templates-text-page-js-1535269777918-d22844cc3df9f359afbe.js","/component---src-layouts-index-js-1535269777918-a020b5fa45586772f790.js"].forEach(function(s){document.write('<script src="'+s+'" defer></'+'script>')})/*]]>*/</script></body>`

Meanwhile the page is rendered on client side and displayed as expected. 
Any ideas what wrong with my integration?

**Comment by arielk5740** [comment]
@btk Yes, it's output of production version (`gatsby build`), not `develop` or `serve`

**Comment by kakadiadarpan** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub issues, we have to clean some of the old issues as many of them have already been resolved with the latest updates.

Please make sure to update to the latest Gatsby version and check if that solves the issue. Let us know if that works for you by adding a comment 👍
